internal/command/root.go

```diff
@@ -75,6 +75,7 @@ func ExecuteContext(ctx context.Context) error {
 	rootCmd.PersistentFlags().StringVar(&opts.WebhookOptions.Password, "webhook-password", "", "webhook password for basic authentication")
 	rootCmd.PersistentFlags().StringVar(&opts.WebhookOptions.Username, "webhook-username", "", "webhook username for basic authentication")
 	rootCmd.PersistentFlags().DurationVar(&opts.WebhookOptions.Timeout, "webhook-timeout", time.Second, "webhook request timeout (zero is no timeout)")
+	rootCmd.PersistentFlags().StringVar(&opts.WebhookOptions.Probe, "webhook-probe", "", "webhook server probe request")
```

Comment:
```suggestion
	rootCmd.PersistentFlags().StringVar(&opts.WebhookOptions.Probe, "webhook-probe", "", "webhook server probe request method")
```

I think if I see "method" in there I would know roughly what the valid options are. I might not get the capitalization correct though (e.g. `--webhook-probe=head` (vs HEAD)).